### DISCUSSION TITLE
Pip install does not include "turbopark_lookup_table.mat"

### DISCUSSION THREAD

**Discussion by mr483** [NatLabRockies/floris#446] [Q&A] [answered]
When trying to use the Turbopark model, I received the error that the file "turbopark_lookup_table.mat" could not be found. Downloading it from Github and adding it in the wake_velocity folder fixed this. I tried installing FLORIS on a new environment and could reproduce the issue, so it seems the file is not included when installing via pip.

**Comment by rafmudaf** [comment, marked as answer]
@mr483 Thanks for pointing this out. I've created an issue for this (#447) and I'll fix it asap.

**Reply by rafmudaf** [reply]
See #447 and [Release v3.1.1 ](https://github.com/NREL/floris/releases/tag/v3.1.1)
